tests/server/blueprints/login/test_views.py

```diff
@@ -51,7 +51,7 @@ def return_user(*args, **kwargs):
     # GIVEN an initialized app with LDAP config params
     with ldap_app.test_client() as client:
         # When submitting LDAP username and password
-        form_data = {"username": "test_user", "password": "test_password"}
```

Comment:
This was wrong, see the parameters passed in the form: https://github.com/Clinical-Genomics/scout/blob/ddd87b6cc437c4e6b71f1bc0bf59b2e5e41a5982/scout/server/blueprints/login/views.py#L65

Who knows what this test was testing before! 🤔